### ISSUE TITLE
6.0.3 version

### ISSUE THREAD

**Issue by jeffreynerona** [dependency-check/azuredevops#83]
this installs 6.0.2, how can we get this release? https://github.com/jeremylong/DependencyCheck/releases/tag/v6.0.3

**Comment by ejohn20** [comment]
Check your **Installer Version** field:

```
{      "name": "dependencyCheckVersion", 
       "type": "string",      
       "label": "Installer Version",      
       "helpMarkDown": "By default, the build task downloads the latest installer from the Dependency Check GitHub releases. Setting this field will change the Dependency Check version downloaded onto the build agent.",      
       "defaultValue": "latest",     
       "required": false    },
```

It should download the latest release by default.

**Comment by jeffreynerona** [comment]
Thanks!. adding the version solved it, I think this pulls 6.0.2 by default https://marketplace.visualstudio.com/items?itemName=dependency-check.dependencycheck

**Comment by ejohn20** [comment]
Perfect, I just ran a dev pipeline with latest and it found the latest release. I think we're OK here as far as I can tell.

```
Downloading Dependency Check latest installer from GitHub..
Downloading ZIP from "https://github.com/jeremylong/DependencyCheck/releases/download/v6.1.6/dependency-check-6.1.6-release.zip"...
...
Dependency-Check Core version 6.1.6
```
